src/aura/STG_CMP_CourseConnections/STG_CMP_CourseConnectionsController.js

```diff
@@ -0,0 +1,40 @@
+({
+  settsLinkClicked : function(component) {
```

Comment:
The controller should delegate to the helper. All (most of) the code should really be in the helper (unless it's so little code that it isn't worth it).